src/VolleyManagement.Backend/VolleyManagement.Domain/GamesAggregate/ResultValidation.cs

```diff
@@ -161,17 +161,9 @@ private static bool IsTechnicalDefeatOptionalSetScoreValid(Score setScore)
 
         private static bool IsOrdinaryRequiredSetScoreValid(Score setScore, int setOrderNumber = 0)
         {
-            bool isValid = false;
-            if (setOrderNumber == Constants.GameResult.MAX_SETS_COUNT)
-            {
-                isValid = IsSetScoreValid(setScore, Constants.GameResult.FIFTH_SET_POINTS_MIN_VALUE_TO_WIN);
-            }
-            else
-            {
-                isValid = IsSetScoreValid(setScore, Constants.GameResult.SET_POINTS_MIN_VALUE_TO_WIN);
-            }
-
-            return isValid;
+            return setOrderNumber == Constants.GameResult.MAX_SETS_COUNT
```

Comment:
If you decided to use ternary operator refactor it even more

Reply:
ok

Reply:
Can you call method once?

Reply:
is it possible?

Reply:
yes)